src/config/config.ts

```diff
@@ -0,0 +1,3 @@
+export const config = {
+  DEFAULT_BANK: '54.183.16.194',
```

Comment:
I believe just an `index.ts` file under the `config` folder would be sufficient?

Also, would be gr8 to make the variable name more explicit, i.e. `DEFAULT_BANK_IP_ADDRESS`

Reply:
did i set it correctly? - tslint is flagging that `@config` has an issue

@jamessspanggg#1464 - idk if i set the config correctly for `@config` alias